server

```diff
@@ -51,6 +53,7 @@ post '/boards' do
   board[:columns] = board[:columns].lines.map(&:strip).reject(&:empty?)
 
   boards[id] = board
+  cookies[:rem_votes] = 3
```

Comment:
yeah that will cause problems, the board post event will change the cookie for who creates the board not everyone that uses it, checkout the previous comment, we won't need this line here.